src/Logging/src/Abstractions/DynamicLoggerProviderBase.cs

```diff
@@ -333,4 +333,12 @@ private IEnumerable<string> GetKeyPrefixes(string name)
 
         return null;
     }
+
+    public void SetScopeProvider(IExternalScopeProvider scopeProvider)
```

Comment:
How or when is this used?

Reply:
For scopes to work, `ConsoleLogger` needs a scope provider [here](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs#L79). It is then subsequently passed on to the formatter [here](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs#L56), which renders the scope values. `LoggerFactory` creates the scope provider and exposes it to logger providers  [here](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Logging/src/LoggerFactory.cs#L197-L202).

Reply:
Oh! I missed the connection with `ISupportExternalScope`